src/contentstack-live-preview-HOC.ts

```diff
@@ -55,10 +55,16 @@ export class ContentstackLivePreview {
         ContentstackLivePreview.subscribers[callbackUid] = callback;
         return callbackUid;
     }
-
+/**
+ * @type {function}
+ * @param onChangeCallback A function param to fetch the data from contentstack database
+ * @param config An optional object param, pass {skipInitRun:Boolean} to skip init call to onChangeCallback
+ * @returns Subscribed Callback UID 
+ */
     static onEntryChange(
-        onChangeCallback: OnEntryChangeCallback
+        onChangeCallback: OnEntryChangeCallback,config?:{skipInitRun?:boolean}
```

Comment:
Let's move the config type into a separate interface and use it here.

Reply:
done